packages/itwin/grouping-mapping-widget/src/widget/components/context/CustomUIProviderContext.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { createContext, useContext } from "react";
+import type { CustomUIProvider } from "../provider/CustomUIProvider";
+
+export const createDefaultCustomUIProvider = (): CustomUIProvider[] => {
+  return [];
+};
+
+export const createCustomUIProvider = (
```

Comment:
what is the point of this??

Reply:
Using context avoids passing in arguments all the way down to grouping component

Reply:
the entire createDefaultCustomuiProvider fn is pointless imo, it just returns an empty array.
feel like this file can bec leaned up further